### PR TITLE
some styling added

### PR DESCRIPTION
**Pull request by elifnurkalkan**

### CODE REVIEW

File: src/views/finalSummaryView.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ export const createFinalSummaryElement = () => {
 
   const element = document.createElement('div');
   element.innerHTML = `
-    <h1>Congratulations ! you answered ${finalScore} from total ${questions.length} questions </h1>
+    <marquee  Scrollamount=15 direction=left>Congratulations ! You answered ${finalScore} correct from  ${questions.length} questions!</marquee>
```
Comment (arkaitzgarro) [inline src/views/finalSummaryView.js:8]: Hey a `marquee` tag! Unfortunately, as much as I love them, they are deprecated and support will be removed from browsers. Please don't use them :(

File: index.html [hunk 2]
```diff
@@ -9,6 +9,44 @@
...
+        width="100%"
+        height="100%"
+        style="position: absolute"
+        frameborder="0"
+        class="giphy-embed"
+        allowfullscreen
+      ></iframe>
+    </div>
+    <p>
+      <a
```
Comment (arkaitzgarro) [inline index.html:26]: All these links can be removed.

**Comment by elifnurkalkan** [comment]
Hi Arkaitz, could you please check it? I am trying to finish styling.

**Review by arkaitzgarro — COMMENTED** [review]
This is fun! The background behaves in a weird way, because I can scroll over it. Can you maybe take a look? If you cannot figure out, it's fine, it looks nice anyway 💅

**Review by arkaitzgarro — APPROVED** [review]

**Pull request merged**